yarn.lock

```diff
@@ -5216,6 +5216,13 @@
   dependencies:
     "@types/node" "*"
 
+"@types/cheerio@^0.22.22":
```

Comment:
we  should dedupe this. you can use [this tool](https://github.com/scinos/yarn-deduplicate) only for this package . ty

Reply:
Did that. Now, the build is failing with `Cannot find name Cheerio` error.
https://github.com/jhipster/generator-jhipster/issues/12593#issuecomment-699449153 suggests upgrading @types/enzyme to resolve the error. Should I do that?

Reply:
done

Reply:
@Hotell I have made all the changes suggested. Please review again.